FHEM/00_SIGNALduino.pm

```diff
@@ -4231,42 +4231,49 @@ sub SIGNALduino_FW_getProtocolList
 	
 	my $id;
 	my $ret;
-	my $s;
-	my $moduleId;
+	my $devFlag = 0;	# 1 - develop version
+	my $devText = "";
+	my %mIdListH = ();
 	my @IdList = ();
 	
+	my $whitelist = AttrVal($name,"whitelist_IDs","#");
+	
+	my $develop = AttrVal($name,"development","");
+	if (length($develop) > 0 && ($develop eq "1" || substr($develop,0,1) eq "y")) {		# develop version
+		$devFlag = 1;
+		$devText = "development version - ";
+	}
+	
+	my $mIdList = join(",",@{$hash->{msIdList}},@{$hash->{muIdList}},@{$hash->{mcIdList}});
+	%mIdListH = map { $_ => 1 } split(",", $mIdList);
```

Comment:
wofür steht das H in mIdList und wieso setzt Du den Wert auf 1. Sind auch andere Werte möglich?

Reply:
H steht für hash, mir ist nichts besseres eingefallen.
Der Wert ist egal da nachher nur mit defined oder exists abgefragt wird

Reply:
Okay verstehe. List und H für Hash irritiert mich da etwas.
Wenn es auch nur darum geht aus den drei Listen einen Hash zu machen, geht das ohne join und split einfacher. Was hälst Du von  activeIdHash als Bezeichner für diese Variable?

https://onlinegdb.com/SkLZfFSgN